packages/cozy-logger/index.d.ts

```diff
@@ -0,0 +1,22 @@
+declare module 'cozy-logger' {
+  declare function log(
+    type: 'debug' | 'info' | 'warn' | 'error' | 'ok' | 'critical',
+    message: string,
+    label?: string,
+    namespace?: string
+  ): void
+
+  declare namespace log {
+    export const addFilter: (filter: unknown) => number
+
+    export const namespace: (
+      namespace: string
+    ) => (type: string, message: string, label?: string, ns?: string) => void
+
+    export const setNoRetry: (obj: unknown) => unknown
+
+    export const setLevel: (lvl: unknown) => void
```

Comment:
so sad to see a lot of unknown

Reply:
Correct but I'm not sure what the types should be unless I spend too much time on it

Reply:
it was a nit sorry